.github/workflows/main.yml

```diff
@@ -64,7 +64,7 @@ jobs:
       - run: sudo apt-get -y update
       - run: sudo apt-get install -y pkg-config libsystemd-dev libdbus-glib-1-dev libelf-dev libseccomp-dev
       - name: Run tests
-        run: cargo test --all --all-features --no-fail-fast
+        run: sudo cargo test --all --all-features --no-fail-fast
```

Comment:
This, unfortunately, does not work, so I will try to find a way through it, otherwise, I will delete the `test_determine_root_path_non_rootless` test.